src/TestFramework/MSTest.Core/Attributes/DataRowAttribute.cs

```diff
@@ -69,7 +69,7 @@ public string GetDisplayName(MethodInfo methodInfo, object[] data)
             {
                 if (data != null)
                 {
-                    return string.Format(CultureInfo.CurrentCulture, FrameworkMessages.DataDrivenResultDisplayName, methodInfo.Name, string.Join(",", data));
+                    return string.Format(CultureInfo.CurrentCulture, FrameworkMessages.DataDrivenResultDisplayName, methodInfo.Name, string.Join(",", (IEnumerable<object>)data));
```

Comment:
Minor: should we consider `AsEnumerable` instead of casting to `IEnumerable<T>`?
https://stackoverflow.com/questions/2013846/why-use-asenumerable-rather-than-casting-to-ienumerablet

Reply:
This is good 👍